pkg/columns/formatter/textcolumns/options.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package textcolumns
+
+type HeaderStyle int
+
+const (
+	HeaderStyleNormal HeaderStyle = iota
+	HeaderStyleUppercase
+	HeaderStyleLowercase
+)
+
+const (
+	DividerSpace = " "
+	DividerTab   = "\t"
+	DividerDash  = "—"
+)
+
+type Option func(*Options)
+
+type Options struct {
+	AutoScale      bool        // if enabled, the screen size will be used to scale the widths
+	ColumnDivider  string      // defines the string that should be used as spacer in between columns (default " ")
+	DefaultColumns []string    // defines which columns to show by default; will be set to all visible columns if nil
+	HeaderStyle    HeaderStyle // defines how column headers are decorated (e.g. uppercase/lowercase)
+	RowDivider     string      // defines the (to be repeated) string that should be used below the header
+	ShowRowDivider bool        // enables/disables the use of the row divider (below header)
+}
+
+func DefaultOptions() *Options {
+	return &Options{
+		AutoScale:      true,
+		ColumnDivider:  DividerSpace,
+		DefaultColumns: nil,
+		HeaderStyle:    HeaderStyleUppercase,
+		RowDivider:     DividerDash,
+		ShowRowDivider: true,
```

Comment:
I `ShowRowDivider` really needed? Isn't it enough to set `RowDivider` to `NoDivider` (To be added)?

Reply:
Removed it! 👍